docs/source/iam.rst

```diff
@@ -4,8 +4,11 @@ IAM in CfnCluster
 ========================
 
 .. warning::
-    Between CfnCluster 1.4.2 and 1.5.0 we added a change to the `CfnClusterInstancePolicy` that adds "ec2:DescribeVolumes" permissions. If you're using a custom policy (e.g. you specify "ec2_iam_role" in your config) be sure it includes this new permission.
 
+    Between CfnCluster 1.5.2 and 1.5.3 we made a change to the CfnClusterInstancePolicy that adds “s3:GetObject” permissions on objects in <REGION>-cfncluster bucket.
+    If you’re using a custom policy (e.g. you specify "ec2_iam_role" in your config) be sure it includes this new permission. See https://cfncluster.readthedocs.io/en/latest/iam.html
```

Comment:
I think we should remove this sentence or adjust it.
If you want to adjust, please move after the "Between CfnCluster 1.4.2 and 1.5.0" sentence, and remove the part "See https://cfncluster.readthedocs.io/en/latest/iam.html" because we are already in this page.